fs_mark/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+FS_MARK_VER="3.3"
+FS_MARK=fs_mark-${FS_MARK_VER}
+PKG_URL="https://github.com/josefbacik/fs_mark/archive/master.tar.gz"
```

Comment:
you're downloading current master branch, not explicit version 3.3 - I see that there is no v3.3 tag/branch and the actual code haven't changed in years - but maybe downloading explicit commit would be better (right now upstream commit might break our port)

https://github.com/josefbacik/fs_mark/archive/2628be58146de63a13260ff64550f84275556c0e.zip